keyboards/crkbd/keymaps/mcrown/mcrown.h

```diff
@@ -0,0 +1,112 @@
+/** @file mcrown.h
+ *  @brief mcrown header thant incluedes key layout and some definintions.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * @author Mario Corona (mariocc@comunidad.unam.mx) 2021
+ *
+ */
+
+#pragma once
+
+#include QMK_KEYBOARD_H
+#include "oled.h"
+
+#define KC_XXXXX KC_NO
+#define KC_LOWER LOWER
+#define KC_RAISE RAISE
+#define KC_CTLTB CTL_T(KC_TAB)
+#define KC_GUIEI GUI_T(KC_LANG2)
+#define KC_ALTKN ALT_T(KC_LANG1)
+
+#define LAYOUT_wrapper(...)   LAYOUT(__VA_ARGS__)
+
+/* For values greater than 0x200, substract that value and add 0x20 so the characters can be mapped  */
+#define RM_LSFT(kc)           ((uint8_t)(0x20)+(kc-0x200))
+
+/* Toggle a boolean variable */
+#define TOGGLE_BOOL_VAR(bv)   (((bv)^true)&true)
+
```

Comment:
```suggestion
```

Reply:
Although that would work I'd rather to keep like this since booleans are no part o C standard and if this part of the code is reuse in a different compiler, that would lead to other issues. Here, we are just toggling a bit (first bit, depending how AVR compiler handles this boolean variables)

Reply:
QMK is only compiled using `gnu99`, which has support for bool: https://github.com/qmk/qmk_firmware/blob/master/tmk_core/rules.mk#L56. There is no need for this overly complex solution just to flip a bit.